src/components/text/TextStyles.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { H1, H2, H3, H4, H5 } from './H'
+import P from './P'
+import Label from './Label'
+import Link from '../navigation/Link'
+import Span from './Span'
+
+storiesOf('Atoms | Text | Headings', module)
+  .add('P', () => <P>Standard Paragraph Style</P>)
+  .add('Small', () => <P small>Small text</P>)
+  .add('Link', () => <Link>Link to another page</Link>)
+  // ^ Link should probably reside in the text directory
```

Comment:
@ric9176 why? I think the primary responsibility of a link is to navigate from one page to another. It's not styling text in my view. Otherwise, we could argue that a TabItem is styled text too, and therefore move the TabItem to the text folder

Reply:
That wasn't my text comment, I'll remove it